tests/unit/landingai/pipeline/test_frameset.py

```diff
@@ -183,6 +192,27 @@ def test_frameset_downsize_bigger_than_original(frame_getter):
     assert get_image_from_frame_or_frameset(frame).size == (width, height)
 
 
+@pytest.mark.parametrize(
+    "frame_getter",
+    [
+        get_frameset,
+        get_frame,
+    ],
+)
+def test_predict_uses_raw_pil_image(frame_getter):
+    """Test that the predict method uses the raw PIL image, not the array-converted image.
+
+    This is to ensure that the image preserves its particularities when sent to predictor.
+    When sending the array-converted image, RGBA images will raise convertion errors, for example.
+    """
+    frame = frame_getter()
+    predictor = mock.Mock()
+    predictor.predict.return_value = []
+    frame.run_predict(predictor)
+    assert predictor.predict.call_count == 1
+    assert predictor.predict.call_args[0][0] is get_image_from_frame_or_frameset(frame)
+
+
```

Comment:
Sorry, I don't understand if this test will try RGB and RGBA images

Reply:
Neither. This test is basically a type test, making sure we will not process the PIL image before sending it to the predictor.

Reply:
Ohhh I got confused... I see that at least we are still using `cereal1.jpeg` . If you can add more images to test this new scenario it would be nice...